rules/parser.y

```diff
@@ -235,6 +237,28 @@ extra_labels_opts  :
                      { $$ = true }
                    ;
 
+vector_matching    :
+                     { $$ = nil }
+                   | GROUP_OP '(' label_list ')'
+                     { 
+                       var err error
+                       $$, err = newVectorMatching("", $3, nil)
+                       if err != nil { yylex.Error(err.Error()); return 1 }
+                     }
+                   | GROUP_MOD GROUP_OP '(' label_list ')'
```

Comment:
Yeah, I don't think GROUP_MOD without the labelnames should be accepted. If a use case comes along we can't solve some other way, we can reconsider.